winui/Rating/Overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# WinUI Rating Overview
+
+The Syncfusion provides the number of stars that represents a rating. And also used to configure the item size, tooltip and the number of displayed items in the SfRating control. The Rating control can be used in various scenarios like movie rating, rating the application and etc.
+
+[!Rating control overview in WinUI](Rating_images/winui_rating_overview.png)
+
+## Key features
+
```

Comment:
@Priyanka-Vijayasankar Use * symbol to show bulletin for representing key features.

Reply:
Used * symbol in the key features.